src/Module.js

```diff
@@ -28,6 +28,7 @@ module.exports = class extends ChainedMap {
         const value = obj[key];
 
         switch (key) {
+          case 'rules':
```

Comment:
Rules doesn't have a high-level API, so I'm not sure why this is needed. All other high-level API interfaces are singular.